### ISSUE TITLE
bug report: error: Could not connect to the server 127.0.0.1:8332

### ISSUE THREAD

**Issue by nimatabatabai** [lightning-power-users/node-launcher#247]
bug report: error: Could not connect to the server 127.0.0.1:8332

Error details:

Node Launcher was updated to v6.0.1 on MacOS. 
_Initially could not locate the Bitcoin.conf file, but I reselected the same folder under settings and it seemed to fix that._ 

Error message in console now: 

> _error: Could not connect to the server 127.0.0.1:8332
> Make sure the bitcoind server is running and that you are connecting to the correct RPC port._

Error message in Bitcoin Output:

> 2019-04-04T20:30:28Z LoadBlockIndexDB(): Block files have previously been pruned
> : You need to rebuild the database using -reindex to go back to unpruned mode.  This will redownload the entire blockchain.
> Please restart with -reindex or -reindex-chainstate to recover.
> 2019-04-04T20:30:28Z : You need to rebuild the database using -reindex to go back to unpruned mode.  This will redownload the entire blockchain.
> Please restart with -reindex or -reindex-chainstate to recover.

Operating System: macOS

**Comment by PierreRochard** [comment]
Yikes! Can you provide the logs from from `~/Library/Application Support/Node Launcher/debug.log`?

**Comment by nimatabatabai** [comment]
[debug.log](https://github.com/lightning-power-users/node-launcher/files/3046745/debug.log)

Here you go! Thanks for looking into this.

**Comment by nimatabatabai** [comment]
If I have my seed words for my LND wallet....can I just reinstall a node and LND wallet from scratch and recover my funds?

**Comment by Scottie1987** [comment]
I was about to open an issue here and found this open issue.

I am having this exact issue with the same error, was there a resolution? 

Were you able to recover your funds? I too have my seed words for the zap wallet and LND wallet. The lightning channel I opened was with the lightning power users

Cheers

**Comment by nimatabatabai** [comment]
Yes, I went back to the last working version of the node launcher and
re-installed that (required a re-download of the blockchain). Then I was
able to recover my wallet in Zap.
---
Best regards,
Nima Tabatabai
